apps/contributions/serializers.py

```diff
@@ -386,15 +390,19 @@ def validate(self, data):
         self.do_conditional_validation(data)
         return data
 
-    def get_bad_actor_score(self, data):
+    def get_bad_actor_score(self, data, action: str = "contribution"):
         """Based on validated data, make a request to bad actor API and return its response."""
         data = data | {
             # we use a PrimaryKeyRelated serializer field for page in BaseCreatePaymentSerializer
             # but BadActorSerializer wants to pk, so we reformat here.
             "page": data["page"].id,
             "referer": self.context["request"].META.get("HTTP_REFERER"),
             "ip": get_original_ip_from_request(self.context["request"]),
+            "action": action,
+            "org": data["page"].organization.id,
```

Comment:
How confident are we about data['page"].organization.id having referential integrity?  Do we have a strong gaurantee here, and if not, what happens if this long chain isn't found at runtime?

Reply:
`data['page']` is a `DonationPage` instance. I'd think a `DonationPage` would always have an `RP` and `Org` associated with it. What do you think?

Or should we do something like:
```suggestion
            "org": data["page"].organization.id if data["page"].organization else None,
```